scripts/container.sh

```diff
@@ -7,6 +7,7 @@ cd "$(dirname "$0")/.."
 
 # normalize branch name to reflect a valid image name
 BRANCH=$(git branch --show-current | sed 's/[^a-z0-9-]/_/ig')
```

Comment:
```suggestion
BRANCH=$(git branch --show-current 2>/dev/null | sed 's/[^a-z0-9-]/_/ig')
```
Let's also hide error messages.